pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -78,7 +78,7 @@ export class JournalOutput {
             <div className="cockpit-logline" role="row" tabIndex="0" key={entry.__CURSOR}
                 data-cursor={entry.__CURSOR}
                 onClick={ev => this.onEvent(ev, entry.__CURSOR, full_content)}
-                onKeyPress={ev => this.onEvent(ev, entry.__CURSOR, full_content)}>
+                onKeyDown={ev => this.onEvent(ev, entry.__CURSOR, full_content)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19083-20230710-104935-c8068a06-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)